build.rs

```diff
@@ -10,6 +10,13 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     println!("cargo:rerun-if-changed=res/proto/google/cloud/texttospeech/v1/cloud_tts.proto");
     println!("cargo:rerun-if-changed=res/proto/google/cloud/texttospeech/v1beta1/cloud_tts.proto");
 
+    let skip_bindgen = env::var("GOOGLE_COGNITIVE_API_SKIP_BINDGEN")
```

Comment:
I would name this differently, maybe **SKIP_PROTO_FILES_COMPILATION**?

also while you want to skip to compile commands for proto compilation you want to preserve **Ok(())**

so rather then calling return you should have two IFs to skip respective code.

can you change accordingly? did you test it works properly (with and without specifying env variable)?